docs/src/pages/[platform]/connected-components/authenticator/quick-start-add.android.mdx

```diff
@@ -0,0 +1,27 @@
+The quickest way to get started is by wrapping your App component (written in Jetpack Compose) with the Authenticator widget.
+Once an end-user has created an account & signed in, your content will be displayed.
+
+Add the Authenticator to your app by wrapping your component (represented here by the `@Composable` function `SignedInContent`) with the `Authenticator` component.
+
+```kotlin
+import com.amplifyframework.ui.authenticator.rememberAuthenticatorState
+import com.amplifyframework.ui.authenticator.ui.Authenticator
+
+class MainActivity : ComponentActivity() {
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+
+    val authenticatorState = rememberAuthenticatorState()
```

Comment:
This call has to be inside the `setContent` lambda. Or since you're not using it for anything just omit entirely (and remove line 20).